components/02-molecules/registration/ENSResolverComponent.tsx

```diff
@@ -65,59 +68,62 @@ export const ENSResolverComponent = ({
         </div>
 
         <div className="flex flex-col border rounded-[8px] border-gray-200 w-full">
-          {/* <div
-            onClick={() => handleENSResolverSelection(radioButtonRefMainnet)}
+          <div
+            onClick={() => handleENSResolverSelection(radioButtonRefDatabase)}
             className={`flex cursor-pointer items-center gap-4 p-3 border-b border-gray-200 ${
               ensResolver === null
                 ? "bg-white"
-                : ensResolver === EnsResolver.Mainnet
+                : ensResolver === EnsResolver.Database
                 ? "bg-[#EEF5FF]"
                 : "bg-white"
             }`}
           >
             <div>
               <RadioButton
-                checked={ensResolver === EnsResolver.Mainnet}
+                checked={ensResolver === EnsResolver.Database}
                 onChange={() => {
-                  setEnsResolver(EnsResolver.Mainnet);
+                  setEnsResolver(EnsResolver.Database);
                 }}
-                ref={radioButtonRefMainnet}
+                ref={radioButtonRefDatabase}
                 label=""
                 name="RadioButtonGroup"
                 value="10"
               />
             </div>
             <div className="flex items-center justify-center gap-2">
-              <EthIcon className="h-6 w-6" />
-              Mainnet
+              <DatabaseIcon className="h-6 w-6" />
+              Off-chain
+              <p className="text-xs mt-1">hosted by blockful</p>
             </div>
-          </div> */}
+          </div>
+
           <div
-            onClick={() => handleENSResolverSelection(radioButtonRefDatabase)}
+            onClick={() =>
+              handleENSResolverSelection(radioButtonRefCustomDatabase)
+            }
             className={`flex cursor-pointer items-center gap-4 p-3 border-b border-gray-200 ${
               ensResolver === null
                 ? "bg-white"
-                : ensResolver === EnsResolver.Database
+                : ensResolver === EnsResolver.Custom
                 ? "bg-[#EEF5FF]"
                 : "bg-white"
             }`}
           >
             <div>
               <RadioButton
-                checked={ensResolver === EnsResolver.Database}
+                checked={ensResolver === EnsResolver.Custom}
                 onChange={() => {
-                  setEnsResolver(EnsResolver.Database);
+                  setEnsResolver(EnsResolver.Custom);
                 }}
-                ref={radioButtonRefDatabase}
+                ref={radioButtonRefCustomDatabase}
                 label=""
                 name="RadioButtonGroup"
                 value="10"
               />
             </div>
             <div className="flex items-center justify-center gap-2">
               <DatabaseIcon className="h-6 w-6" />
-              Off-chain
-              <p className="text-xs mt-1">hosted by blockful</p>
+              Custom Off-chain Resolver
```

Comment:
I'm not sure if the custom resolver is necessarily off-chain, maybe it's better to keep it just "Custom Resolver"

Reply:
Nice! Great catch